lib/navbar/theme.module.scss

```diff
@@ -15,8 +16,10 @@
 }
 :local(.innerWrapper) {
   display: flex;
+  padding: 1% 3%;
```

Comment:
- `padding: 0% 2%;`

Reply:
can provide padding values in pixels if needed.


Menu Icon:

Reduce Size 
Add padding if required.
Reduce right margin, already enough space between icon and Title

Reply:
Can use below image for reference

Before:
<img width="994" alt="screenshot 2018-12-31 at 2 17 24 pm" src="https://user-images.githubusercontent.com/15786884/50557019-d1109300-0d06-11e9-93be-02206dd26ee1.png">





After:
<img width="920" alt="screenshot 2018-12-31 at 2 17 17 pm" src="https://user-images.githubusercontent.com/15786884/50557020-d1a92980-0d06-11e9-9b1d-c9e0804630df.png">

Reply:
@kaushiknishchay the menu icon is just an example logo @KRRISH96 had put in. The actual menu icon only appears on smaller screen ( at the right end of the navbar ). I'm updating the navbar usage demo code now to avoid any confusion in the future.

Reply:
Removed this hamburger menu svg as logo and added lite ui logo instead. https://github.com/Codebrahma/react-lite-ui/pull/297/commits/cf4368b1f14053cc0305a5f030c91b0d55674989